### PR TITLE
(fix) (UMCS-434) Make examples use basket v2.

### PR DESCRIPTION
**Pull request by Ryouzanpaku**

### CODE REVIEW

**Review by sixer1182 — COMMENTED** [review]
Will this be mentioned in the changelog?

File: src/Resources/TransactionTypes/Authorization.php [hunk 1]
```diff
@@ -280,7 +280,7 @@ public function cancel($amount = null): Cancellation
     /**
      * Charge authorization.
      *
-     * @param null $amount
+     * @param float $amount
```
Comment (sixer1182) [inline src/Resources/TransactionTypes/Authorization.php:283]:
```suggestion
     * @param float|null $amount
```

**Review by sixer1182 — COMMENTED** [review]
see comment

File: src/Resources/TransactionTypes/Authorization.php (hunk 1 above)
Comment (Ryouzanpaku) [reply src/Resources/TransactionTypes/Authorization.php]: I adjusted this, and for the rest of that class to make it consistent for that class.

**Pull request merged**